### ISSUE TITLE
underutilized node not getting evict 

### ISSUE THREAD

**Issue by hjavaid06** [aws/karpenter#1829]
**Is an existing page relevant?**

yes 
**What karpenter features are relevant?**

Deprovisioning

Actually, my question or issue is that when the load on any node is reduced e.g only one pod running on a node that was provisioned by karpenter previously  , is not getting evicted as it is underutilized as  only one pod is running on it 
so is there any mechanism that moves that pod to another node (which has a capacity)  and deleted the new node  ?

in simple terms Reallocating the pods in nodes for better resource utilization

i hope made my point clear , let me know if there is any ambiguity

**Comment by suket22** [comment]
I'll close this out since this is better tracked via #1091. Let us know if there's anything else we can help with.
